src/Employee.java

```diff
@@ -0,0 +1,62 @@
+import java.util.Objects;
+
+public class Employee {
+    private String fullNameEmployee;
+    private int departmentNumberEmployee;
+    private int salaryEmployee;
+    static int enumerator=1;
```

Comment:
```suggestion
    private static int enumerator=1;
```
все поля должны быть приватные